std_msgs/Header  header
uint32 Gear                           # 档位
float32 ThrottlePedal                 # 油门踏板开度
float32 BrakePedal                    # 制动踏板开度
float32 SteeringAngle                 # 方向盘转角
